screens/Login.tsx

```diff
@@ -0,0 +1,85 @@
+import { StyleSheet, TextInput, Text, View, Button } from 'react-native';
+import Form from 'react-native-form';
```

Comment:
As a general rule, we want to avoid adding new packages when we don't _have_ to.

Also, you wanna make sure they appear credible, up-to-date, and well-maintained. 

For example, this `[react-native-form](https://www.npmjs.com/package/react-native-form)` hasn't been updated since 2018, so it doesn't appear to be regularly maintained. That meant incompatibility or other bugs in the dated package could cause problems for our app down the road. Also, it's not very widely used (612 weekly downloads is pretty low), which is also another yellow/red flag. 

What did you want to use the `Form` component for that raw TextInputs, Views, and Buttons can't handle? If there are additional things you think we need, we could look into finding a more credible package to fit those needs.